karafka.gemspec

```diff
@@ -5,6 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'karafka/version'
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
sidenote: really don't like the inline disabling - will disable this for rubocop settings in coditsu (but for not it can stay)

Reply:
We should drop this disable, `Metrics/BlockLength` should not be run for `*.gemspec`